sormas-ui/src/main/java/de/symeda/sormas/ui/UserProvider.java

```diff
@@ -97,6 +97,11 @@ public boolean hasNationJurisdictionLevel() {
 		return getJurisdictionLevel() == JurisdictionLevel.NATION;
 	}
 
+	public boolean isAdmin() {
```

Comment:
We should probably search for `I18nProperties.getEnumCaption(role)` as the caption will be translated to the server's locale during migration

Reply:
Due to the point that I'm streaming all userRoles, and then looking by caption, this method will not work. `user.getUserRoles().stream().peek(i->i.getCaption().contains(I18nProperties.getEnumCaption(DefaultUserRole.ADMIN))).collect(Collectors.toList())` will return  2 instead of 1, in case if 1 other user still have ADMIN role. And in this case, user which is not authorized to link elements, will be able to do this. 
Due  to that, I'm still thinking that current approach is the best  one :) 
![image](https://user-images.githubusercontent.com/6985315/172753653-9fdac84e-4a67-4032-bc75-af2b975581ed.png)